pkg/plugins/models.go

```diff
@@ -18,16 +18,16 @@ type PluginNotFoundError struct {
 }
 
 func (e PluginNotFoundError) Error() string {
-	return fmt.Sprintf("plugin with ID %q not found", e.PluginID)
+	return fmt.Sprintf("plugin with ID %s not found", e.PluginID)
 }
 
 type DuplicatePluginError struct {
-	Plugin         *PluginBase
-	ExistingPlugin *PluginBase
+	PluginID          string
+	ExistingPluginDir string
 }
 
 func (e DuplicatePluginError) Error() string {
-	return fmt.Sprintf("plugin with ID %q already loaded from %q", e.Plugin.Id, e.ExistingPlugin.PluginDir)
+	return fmt.Sprintf("plugin with ID %s already exists in %s", e.PluginID, e.ExistingPluginDir)
```

Comment:
Same comment as before, I think changing to unquoted strings is a change for the worse, since it makes debugging potentially more difficult.